utilities/test_suite/HIP/testAllScript.sh

```diff
@@ -311,8 +328,13 @@ else
                 printf "\n\n\nRunning New Bit Depth...\n-------------------------\n\n"
                 for ((outputFormatToggle=0;outputFormatToggle<2;outputFormatToggle++))
                 do
-                    SRC_FOLDER_1_TEMP="$SRC_FOLDER_1"
-                    SRC_FOLDER_2_TEMP="$SRC_FOLDER_2"
+                    if [ "$case" -eq "82" ]; then
+                        SRC_FOLDER_1_TEMP="$RICAP_SRC_FOLDER"
```

Comment:
We might need to make this change only when QA mode is enabled
in case of general unit test mode for RICAP, user is expected to pass any folder having images of same size
inside the .cpp please add error handling to see if all images in folder are of same size, incase of RICAP to avoid failure cases and mention dataset passed should be of same size

if user does not pass any input_path, we can the folder path set to RICAP 3 image folder